recipes/wub/meta.yaml

```diff
@@ -47,7 +47,7 @@ requirements:
   - tqdm
   - h5py
   - statsmodels
-  - htslib <1.4
+  - htslib {{ CONDA_HTSLIB }}*
```

Comment:
When we remove `htslib` from the `build` requirements, we don't have to pin it anymore. If the previous restriction on `<1.4` still applies, that should be restored. If there are no restrictions regarding the `htslib` version, then
```yaml
    - htslib
```
should suffice.
Also there seems to be a version constraint in https://github.com/nanoporetech/wub/blob/v0.2.0/setup.py#L19 for `pandas` -- we should add that one here, too, i.e.,
```yaml
    - pandas >=0.20.2
```